### DISCUSSION TITLE
请问 t-class 的作用是什么？

### DISCUSSION THREAD

**Discussion by iwanglian** [Tencent/tdesign-miniprogram#1432] [Q&A] [answered]
最近在看项目代码，有一个疑问，很多组件都使用 t-class 作为 css选择器，但代码中没看到哪里有定义 t-class
比如button
`export default class Button extends SuperComponent {
  externalClasses = [`${prefix}-class`, `${prefix}-class-icon`, `${prefix}-class-loading`];
` 

`<button
  style="{{ customStyle }}"
  data-custom="{{ customDataset }}"
  class="{{className}} {{prefix}}-class"`

包括 t-class-icon,  t-class-loading 也都没定义，这样设计的目的是什么？

**Comment by LeeJim** [comment, marked as answer]
`${prefix}-class` 即是 t-class
